src/Compilers/CSharp/Test/Symbol/Symbols/DefaultInterfaceImplementationTests.cs

```diff
@@ -17332,10 +17332,10 @@ int I2.P6
                                                  targetFramework: TargetFramework.NetStandardLatest);
             Assert.True(compilation1.Assembly.RuntimeSupportsDefaultInterfaceImplementation);
 
-            // PROTOTYPE(DefaultInterfaceImplementation): The wording "accessor not found in interface member" is somewhat misleading
-            //                                            in this scenario. The accessor is there, but cannot be implemented. Prehaps
-            //                                            the message should be adjusted. Should also check diagnostics for an attempt
-            //                                            to implement other sealed members.
+            // https://github.com/dotnet/roslyn/issues/34455: The wording "accessor not found in interface member" is somewhat misleading
+            //                                                in this scenario. The accessor is there, but cannot be implemented. Prehaps
```

Comment:
>Prehaps [](start = 130, length = 7)

Typo

Reply:
> Typo

This is preexisting condition, the entire comment will be removed once the referenced issue is addressed.